data/events/scripts/player.lua

```diff
@@ -253,9 +253,9 @@ function Player:onMoveItem(item, count, fromPosition, toPosition, fromCylinder,
 			return false
 		end
 		exhaust[playerId] = true
-		addEvent(function()
+		addEvent(function(playerId)
```

Comment:
```suggestion
		addEvent(function(playerParamId)
```

Reply:
> If you removed it below, you could also remove it here.

I thought I had presented Luan's suggestion, I'll do it now.

Reply:
```suggestion
		addEvent(function(playerIdInEvent)
			exhaust[playerIdInEvent] = false
		end, 2000, playerId)
```

Reply:
preciso pa

> 

preciso passar uma variavel que exista. onde está essa variavel `playerIdInEvent` ?